app/src/main/java/de/westnordost/streetcomplete/quests/camping/AddCampType.kt

```diff
@@ -0,0 +1,42 @@
+package de.westnordost.streetcomplete.quests.camping
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.OUTDOORS
+import de.westnordost.streetcomplete.osm.Tags
+import de.westnordost.streetcomplete.util.ktx.toYesNo
+
+class AddCampType : OsmFilterQuestType<CampType>() {
+
+    override val elementFilter = """
+        nodes, ways with
+          tourism = camp_site
+          and (!caravans or !tents)
+          and !backcountry
+    """
+    override val changesetComment = "Survey who may camp here"
+    override val wikiLink = "Key:caravans"
+    override val icon = R.drawable.ic_quest_camp_type
+    // you can often see caravans and/or tents from the outside, so usually there is no need for this quest to be disabled by default
```

Comment:
but lack of caravans right now is not answering question whether caravans are allowed or not

Reply:
A bit like the seating quests (or pedestrian crossing audible indication), answering about presence is far easier than absence!

Reply:
Still, I think this is fine. Camp sites are open areas usually, so you don't really need to go "inside" the way you would do for shops etc.